doc/docusaurus/.gitignore

```diff
@@ -18,3 +18,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+package-lock.json
```

Comment:
```suggestion

package-lock.json
yarn.lock
```
?

Reply:
Let's keep yarn lockfile. JIC we need to debug some dependency issues (sincerely hope not)

The reason for removing `package-lock.json` was that we should stick to a single dependency management tool, and since docusaurus recommends `yarn` -- let's just keep only that.